client/signup/main.jsx

```diff
@@ -23,6 +23,7 @@ import {
 } from 'lodash';
 import { translate } from 'i18n-calypso';
 import { connect } from 'react-redux';
+import moment from 'moment';
```

Comment:
Since `moment` is used only in the call to `isUserRegistrationDaysWithinRange`, which in turn isn't used anywhere else in the codebase, can we try to refactor this a bit so we don't add the whole `moment` library here? 

Possibly adding a new selector that will check the target user for the current date or something like this? 

As `isUserRegistrationDaysWithinRange` isn't used anywhere we can update it to do that or add a parameter to do that. For example if the `moment` parameter is null to accept `current time` and do calculation accordingly.

Reply:
> As isUserRegistrationDaysWithinRange isn't used anywhere we can update it to do that or add a parameter to do that.

Ah, great idea. Thank you.

Reply:
@bisko wrote...

> As isUserRegistrationDaysWithinRange isn't used anywhere we can update it to do that or add a parameter to do that. For example if the moment parameter is null to accept current time and do calculation accordingly.

I updated. Thanks again!